### DISCUSSION TITLE
Can someone help me understand vertdispratio parameter for repair_non_manifold_vertices_by_splitting?

### DISCUSSION THREAD

**Discussion by Cervidellus** [cnr-isti-vclab/PyMeshLab#100] [Q&A] [answered]
I am trying to clean up some meshes with lot's of non-manifold things.  repair_non_manifold_vertices_by_splitting, and ideally the vertices that are created should be displaced some small amount, so that a remove_duplicate_vertices doesn't just merge them back together. I am assuming that that is what vertdispratio does? But can someone help me understand what would make sense to pass to it? Default is zero, does 10 make sense? 1?

**Comment by cignoni** [comment, marked as answer]
Solved in https://github.com/cnr-isti-vclab/meshlab/commit/f61a72f1f6ef31ea05ef2f18e01b33388bc5eab1  
Improved the description of the parameter now it says: 
This parameter denote the ratio ⍺ of displacement of a vertex. When a vertex <i>v</i> is split, it is moved towards the barycenter <i>b</i> of the FF connected faces sharing it of a (<i>v</i>-<i>b</i>)*⍺. When ⍺ is zero vertex is not displaced. When ⍺ is 0.5 the new vertex is half away toward the barycenter of the face. Reasonable values are in the [0 .. 0.1] range.
